site/tests/integration/block-publish-npm.test.ts

```diff
@@ -1,118 +1,121 @@
-import { Page } from "playwright";
-import { expect, test } from "playwright-test-coverage";
+import { test } from "playwright-test-coverage";
 
-import { publishBlock } from "../shared/blocks";
-import { resetDb } from "../shared/fixtures";
-import { login } from "../shared/nav";
+test("TODO: uncomment this file when NPM_PUBLISHING feature is released", () => {});
```

Comment:
We can use [`test.skip`](https://playwright.dev/docs/test-annotations#skip-a-test) instead.

Lines we’ve commented out can still be linted. Leaving them as comment will produce inaccurate git blame when it's time to restore the test.

Reply:
I thought something like test.skip, based on the env variable, so when we enable the env variable flag, tests would start running again right? @CiaranMn

Reply:
If we stop running the tests there is a chance we make some changes that break the feature without knowing, and then when we want to actually enable it we have to figure out what they were.

If the tests don't take long to run I don't see the advantage in keeping them running (by setting the environment variable for the tests)

Reply:
Possibly it's the 'can't publish the same npm-linked package twice', since this is only prevented if running in production. You could also make that check apply in test mode (or only skip it in development mode)

Reply:
Hmm, I think it could be sth else, since it's calling `resetDb` in `beforeEach` on every test. Btw, I'm not sure if it was `can't publish the same npm-linked package twice`, need to check